client/src/utils/UIComponents.js

```diff
@@ -198,12 +198,10 @@ class Pagination extends Component {
       innerClass={className}
 
       // Some defaults for the styling
-      prevPageText={"Previous"}
-      nextPageText={"Next"}
       itemClass={"page-item"}
       linkClass={"page-link"}
       activeClass={"page-item active"}
-      hideFirstLastPages={false}
+      hideFirstLastPages={true}
```

Comment:
Sorry, I think there was a misunderstanding in the design. The switch to "<" and ">" for previous/next is great, but the first and last page buttons should stay.

```suggestion
      hideFirstLastPages={false}
```